packages/phone-number-privacy/signer/src/server.ts

```diff
@@ -60,35 +58,25 @@ export function startSigner(
     res.send(PromClient.register.metrics())
   })
 
-  const addEndpointWithTimeout = (
+  const addEndpoint = (
     endpoint: SignerEndpoint,
     handler: (req: Request, res: Response) => Promise<void>
   ) =>
     app.post(endpoint, async (req, res) => {
       const childLogger: Logger = res.locals.logger
-      const timeoutRes = Symbol()
       try {
-        // TODO(2.0.0, timeout) https://github.com/celo-org/celo-monorepo/issues/9845
-        await timeout(handler, [req, res], config.timeout, timeoutRes)
+        await handler(req, res)
       } catch (err: any) {
         // Handle any errors that otherwise managed to escape the proper handlers
-        let errorMsg: string = ErrorMessage.UNKNOWN_ERROR
-        let errToLog = err
-        if (err === timeoutRes) {
-          Counters.timeouts.inc()
-          errorMsg = ErrorMessage.TIMEOUT_FROM_SIGNER
-          errToLog = new Error(errorMsg)
-        }
-        childLogger.error({ errToLog })
-        if (!res.writableEnded) {
+        childLogger.error({ err }, 'Caught error in outer endpoint handler')
```

Comment:
I left a comment in the combiner about adding an ErrorMessage so we can enable log based metrics here. It would be good to log that same error message here, even though the signer monitoring setup doesn't use log based metrics. Could you also create a prometheus metric so we can alert on this for the signer as well?

Reply:
yup good idea!